api/models.py

```diff
@@ -79,6 +80,23 @@ class Config:
             ObjectId: str,
         }
 
+    @classmethod
+    def serialize_paginated_data(cls, data: list):
+        """
+        Serialize models to generate response without using alias.
+        This is required to get models with `id` field in the response
+        instead of `_id`.
+        In usual cases providing `response_model_by_alias` to endpoint
+        definition serves the purpose. However, that doesn't work in case
+        of paginated data. Hence, need to serialize it manually.
+        """
+        serialized_data = []
+        for obj in data:
+            serialized_data.append(
+                jsonable_encoder(cls(**obj), by_alias=False)
```

Comment:
How about `obj.dict()` or `obj.json()`?

Reply:
Also I think this should probably be outside of `api.models` so we don't add a dependency on `fastapi`.  Maybe it could be in `main` instead as that's where FastAPI is actually used.

Reply:
Why would you prefer to do what FastAPI is doing in its underlying implementation rather than the method provided by the object?

Actually I guess it should be something like this:
```python
for kwargs in data:
    obj = cls(**kwargs)
    serialised_data.append(obj.dict(by_alias=False))
```
The pagination logic isn't really class-specific so this could be a standalone function with a model argument to know the class to call to make the objects, assuming all the objects are of the same type.

Reply:
> Why would you prefer to do what FastAPI is doing in its underlying implementation rather than the method provided by the object?
> 
> Actually I guess it should be something like this:
> 
> ```python
> for kwargs in data:
>     obj = cls(**kwargs)
>     serialised_data.append(obj.dict(by_alias=False))
> ```
> 
> The pagination logic isn't really class-specific so this could be a standalone function with a model argument to know the class to call to make the objects, assuming all the objects are of the same type.

Yes, I agree that we should reduce dependency.
I dropped the encoder from fastapi.
It turns out that `cls(**obj).dict()` works fine as by default `by_alias` flag is set to `False`.

Reply:
> 

I think the serialization should be specific to models.
It's possible that the various models will require serialization differently based on their fields.

Reply:
It's one thing to add the possibility to override how serialisation is done in each model, and another one to add a standard option for not using aliases which is what we're doing here.

Reply:
OK. Updated the code.